.circleci/continue_config.yml

```diff
@@ -454,6 +447,10 @@ workflows:
           matrix:
             parameters:
               python_version: ["3.7", "3.8", "3.9", "3.10"]
+          pre-steps:
+          - run:
+              name: Force install GPL dependency (unidecode)
+              command: echo 'export AIRFLOW_GPL_UNIDECODE=yes' >> $BASH_ENV
```

Comment:
Are we sure this is the right choice? I think this plugin meets the terms of the GPL license but I'm not certain. As I understand it we have an option here to use `text-unidecode` which is not distributed with a GPL license.

Reply:
If you scroll up the commit history, I did try it in https://github.com/kedro-org/kedro-plugins/pull/52/commits/4bb506ac6d69c612e0ce9f307ab55130f045fc66, but it's quite possible something else was going on then. Let me see if it works to change it.

I don't think it's a big deal though, since we're only using it for testing, but guess we may as well avoid it if it works.

Reply:
Ok, seems it works!